docs/src/index3dae.md

```diff
@@ -0,0 +1,34 @@
+#### Fun-Fact: Solving an Index-3 DAE
+The following is based on the
+["Automatic Index Reduction of DAEs"](https://mtk.sciml.ai/stable/mtkitize_tutorials/modelingtoolkitize_index_reduction/)
+tutorial by
+[ModelingToolkit.jl](https://mtk.sciml.ai/stable/), which demonstrates how the classic `Rodas4` solver fails to solve a DAE due to the fact that it is of index 3; which is why ModelingToolkit's automatic index reduction is so useful.
+It turns out that __our probabilistic numerical solvers can directly solve the index-3 DAE__:
```

Comment:
I assume that requires a consistent initialization?

Reply:
Yes I'd think so. But I didn't look into higher indices enough so I didn't plan to link to that part of the documentation yet. I was mostly just intrigued that our solver didn't just crash (which I would have expected it to do).